README.md

```diff
@@ -263,7 +288,7 @@ IR for all the target backends:
 
 To turn on IR transformations set the following properties in your `gradle.properties` file:
 
-> Please note, that starting from version `0.24.0` of the library your project is required to use `Kotlin version >= 1.9.0`. 
```

Comment:
Here, the version should remain 0.24.0, innit?

Reply:
Thank you! I didn't notice that